barretenberg/cpp/src/barretenberg/commitment_schemes/kzg/kzg.hpp

```diff
@@ -82,7 +82,7 @@ template <typename Curve> class KZG {
 
         GroupElement P_0;
         if constexpr (Curve::is_stdlib_type) {
-            auto builder = verifier_transcript->builder;
+            auto builder = quotient_commitment.get_context();
```

Comment:
we no longer store the builder in the transcript, so I get the builder from another stdlib type.